distributions/kfdef/servicemesh/subscriptions.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  name: smsubscriptions
+  namespace: openshift-operators
+spec:
+  applications:
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: openshift-service-mesh/
+      name: openshift-service-mesh
+  repos:
+    - name: manifests
+      uri: https://github.com/VaishnaviHire/odh-manifests/archive/integrate_service_mesh.tar.gz
+  version: v1.3.0-tag
```

Comment:
Why do we need a new kfdef and a new folder, service mesh, under the distribution folder? We need to move this, the kfdef folder is only for example kfdefs that install an example platform

Reply:
We would need to wait for the Servicemesh operators to complete the installation, hence a new kfdef. I added the ServiceMesh folder to have the two required Kfdef( subscriptions kfdef + applications kfdef) examples.